src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.AbstractIntroduceVariableCodeAction.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.CodeAnalysis.IntroduceVariable;
 
 internal partial class AbstractIntroduceVariableService<TService, TExpressionSyntax, TTypeSyntax, TTypeDeclarationSyntax, TQueryExpressionSyntax, TNameSyntax>
 {
-    internal abstract class AbstractIntroduceVariableCodeAction : CodeAction
+    private abstract class AbstractIntroduceVariableCodeAction : CodeAction
```

Comment:
merge of https://github.com/dotnet/roslyn/pull/73392 into this PR.  once that goes in, this will disappear from this PR.